components/brave_rewards/resources/shared/components/newtab/stories/index.tsx

```diff
@@ -24,50 +24,64 @@ function actionLogger (name: string, ...args: any[]) {
   return (...args: any[]) => console.log(name, ...args)
 }
 
+const style = {
```

Comment:
Is this a pattern in rewards? Never seen it before, but its cool!

Reply:
My eyes prefer the lower-cased, namespaced names for style-only elements, and in the rewards stuff we do:

```js
import * as style from './this_components.style'
```

so this is just mimicking that without a different style module.